ui/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -150,6 +150,7 @@ export default class SendAssetRow extends Component {
       if (token) {
         return this.renderToken(token);
       }
+      return this.renderToken(details);
```

Comment:
I am bit confused by code above. Why we can not always use `return this.renderToken(details);`.

Reply:
My best guess is when we add tokens to the store, we use API to discover more information about the token that might not be present in details. 

Of course we could remove lines 147:152 and this PR still works fine. I'm just not sure what other parts of the code might break. If we do remove those lines, we need to make sure we do proper regression.